infrastructure/modules/service-bus/output.tf

```diff
@@ -1,7 +1,7 @@
 output "queues_default_primary_connection_string" {
-  value = ["${azurerm_servicebus_queue_authorization_rule.service_bus_queue_auth_rule.*.primary_connection_string}"]
+  value = ["azurerm_servicebus_queue_authorization_rule.service_bus_queue_auth_rule.*.primary_connection_string"]
```

Comment:
I think I made a mistake here. The quotes should probably be removed. As it stands the output is just the string (i.e. not resolved):
```
service_bus_queue_keys = [
  "azurerm_servicebus_queue_authorization_rule.service_bus_queue_auth_rule.*.primary_key",
]
```